Magento_Sales/templates/order/history.phtml

```diff
@@ -107,14 +107,14 @@
                                 data-th="<?= $block->escapeHtmlAttr(__('Order Total')) ?>"
                                 role="gridcell"
                             >
-                                <?= $block->stripTags($_order->formatPrice($_order->getGrandTotal())) ?>
+                                <?= $block->escapeHtml($block->stripTags($_order->formatPrice($_order->getGrandTotal()))) ?>
```

Comment:
we need to remove escapeHtml and stripTags and add noEscape like in original file